### ISSUE TITLE
Brave app is not working properly on Redmi 3 with MIUI 7.2 stable

### ISSUE THREAD

**Issue by MysteryE** [brave/browser-android#753]
My device data:
Redmi 3
ROM: MIUI 7.2 stable 7.2.7.0 LHPCNDB
Running on Android 5.1.1 LMY47V

Brave version: 1.9.19

Problems:
1. Right after install the app, launch the app, I can't even click the "OK" button when this message appear "By continuing, you acknowledge bla bla bla.......". I have to close the app and re-open, then I'm able to click the OK button and continue to the homepage of the app.
2. Click "New Bubble" button, the keyboard appears. But nothing else shown, not even the url that I typed. When I press back button, the keyboard now gone and back to homepage but I can't click anything, like the buttons or the settings icon, anything. The app froze? Close (press back 2 times or just a single press of home button) and re-open the app solve this froze issue. But the "New Bubble" button still doesn't work.

Basically, I can't do a thing with Brave.
Please help.

**Comment by srirambv** [comment]
Similar issue reported by a friend who's on Redmi Note 3, MIUI 8.0.1, running Android 5.1.1 LMY47V. App hangs every time a new bubble is tapped.

**Comment by SergeyZhukovsky** [comment]
Hi @MysteryE, @srirambv . Could you please check that you have that permission on, for Brave on your device?
You should grant the permission to pop UP windows.

**Comment by srirambv** [comment]
@SergeyZhukovsky  Confirmed working after enabling Allow notifications.
